src/encoded/types/__init__.py

```diff
@@ -247,7 +247,7 @@ def unique_keys(self, properties):
         "type": "string",
```

Comment:
this needs to change to "type": "integer"

Reply:
Oops. Fixed this.